.circleci/config.yml

```diff
@@ -1178,6 +1178,8 @@ jobs:
       - checkout
       - react-get-deps
       - run: make -C webui/react test
+      - store_test_results:
+          path: webui/react/coverage
```

Comment:
**suggestion:** there is an HTML report that is generated at `webui/react/coverage/lcov-report/` (including entry point of `index.html`). wdy think about adding:
```
- store_artifacts:
    path: webui/react/coverage/lcov-report
```

Reply:
https://circleci.com/docs/2.0/code-coverage/#javascript from circleci suggests the same as well